draft-ietf-jsonpath-base.md

```diff
@@ -555,7 +581,7 @@ of the input node.
 The result is a list of three nodes: `{"b":0}`, `{"b":1}`, and `{"c":2}`.
 
 Finally, `.b` selects from any input node of type object with a member name
-`b` and selects the value of the input node corresponding to that name.
+`b` and selects the node of the member value of the input node corresponding to that name.
```

Comment:
I have a hard time understanding what a node of a value might be. This seems to be dimensionally wrong. Maybe there are too many objects in that sentence?

Reply:
I agree that this is rather verbose now.  Maybe

```suggestion
Finally, `.b` selects from any input node of type object any node corresponding to
a member named `b`.
```